src/main/java/staffconnect/logic/LogicManager.java

```diff
@@ -62,8 +62,8 @@ public CommandResult execute(String commandText) throws CommandException, ParseE
     }
 
     @Override
-    public ReadOnlyAddressBook getAddressBook() {
-        return model.getAddressBook();
+    public ReadOnlyStaffBook getStaffBook() {
+        return model.getStaffBook();
```

Comment:
I think covcode is refering to these not changed in the test files, e.g this one should be referring to src/test/java/staffconnect/model/ModelManagerTest.java